atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraConstants.java

```diff
@@ -50,9 +50,9 @@
     static final String DEFAULT_RACK = "rack1";
     static final String SIMPLE_RF_TEST_KEYSPACE = "__simple_rf_test_keyspace__";
     static final String REPLICATION_FACTOR_OPTION = "replication_factor";
-    // 4 days; Hinted-Handoffs MUST expire well within this period for delete correctness
+    // 1 hour; Hinted-Handoffs MUST expire well within this period for delete correctness
     // (I believe we will be expiring hints in half this period)
```

Comment:
Hinted handoff expiration is a cluster level setting in cassandra.yaml, not a CF level setting. For our internal distribution that expiration is current set at 20 minutes by default.